packages/react-accordion/src/components/Accordion/index.ts

```diff
@@ -1,5 +1,16 @@
 export * from './Accordion';
-export * from './Accordion.types';
+// Explicit exports to ommit AccordionCommon
+export type {
+  AccordionIndex,
+  AccordionToggleEvent,
+  AccordionToggleEventHandler,
+  AccordionContextValue,
+  AccordionContextValues,
+  AccordionSlots,
+  AccordionToggleData,
+  AccordionProps,
+  AccordionState,
+} from './Accordion.types';
```

Comment:
nit: order these alphabetically
```suggestion
export type {
  AccordionContextValue,
  AccordionContextValues,
  AccordionIndex,
  AccordionProps,
  AccordionSlots,
  AccordionState,
  AccordionToggleData,
  AccordionToggleEvent,
  AccordionToggleEventHandler,
} from './Accordion.types';
```